draft-jholland-quic-multicast.md

```diff
@@ -187,6 +187,16 @@ The server maintains a full view of the traffic received by the client via the A
 
 Under sustained persistent loss, the server SHOULD instruct the client such that the aggregate rate of joined sessions remains under the data rate successfully received by the client in the recent past.
 
+# Data Integrity
+
+TODO: import the {{I-D.draft-krose-multicast-security}} explanation for why extra integrity protection is necessary (many client have the shared key, so AEAD doesn't provide authentication against other valid clients on its own).
+
+## Packet Hashes {#packet-hashes}
```

Comment:
This TODO should probably wait until AMBI went through last call I think? Any security reviews for AMBI would obviously also be very helpful for this. I also opened another [issue](https://github.com/GrumpyOldTroll/ietf-dorms-cluster/issues/11) recently that might become relevant here eventually (as its specifically concerning the hash algorithm).

Reply:
Yes, something like that. Ultimately I think a decent description of what this is protecting against probably belongs somewhere in this doc, so I thought I'd leave a placeholder.  Agreed there's a good chance reviews on AMBI will likely impact this.